unity-ggjj/Assets/Scripts/AudioController/AudioController.cs

```diff
@@ -67,22 +66,11 @@ public void PlaySong(AudioClip song, float transitionTime)
     /// Creates a child object that has an AudioSource component
     /// </summary>
     /// <returns>The AudioSource of the new child object</returns>
-    private AudioSource CreateAudioSource(string gameObjectName)
-    {
-        var newGameObject = new GameObject(gameObjectName);
-        newGameObject.transform.parent = transform;
-        return newGameObject.AddComponent<AudioSource>();
-    }
-
-    /// <summary>
-    /// Plays sound effect of desired name.
-    /// </summary>
-    /// <param name="sfx">Name of sound effect asset, must be in `Resources/Audio/SFX`</param>
-    public void PlaySfx(string sfx)
-    {
-        AudioClip soundEffectClip = _narrativeGameState.ObjectStorage.GetObject<AudioClip>(sfx);
-        PlaySfx(soundEffectClip);
-    }
+    private AudioSource CreateAudioSource(string gameObjectName) => new GameObject(gameObjectName) {
+            transform = {
+                parent = transform
+            }
+        }.AddComponent<AudioSource>();
```

Comment:
Personally I feel this reads a little confusingly with a lambda. At first glance I thought you were passing the lambda as an argument. Would prefer the following:
```suggestion
    private AudioSource CreateAudioSource(string gameObjectName)
    {
        return new GameObject(gameObjectName)
        {
            transform =
            {
                parent = transform
            }
        }.AddComponent<AudioSource>();
    }
```

Reply:
Actually just updated it based on @lyndsiWilliams comments before I saw your review! 🙈

[Is this new format](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/blob/feature/178-actor-slots-by-name/unity-ggjj/Assets/Scripts/AudioController/AudioController.cs#L69-L72) an okay middle-ground for you or would you like to still see some changes to it?

Reply:
probably could just be one-lined with a lambda in the new format. It's just having the open parenthesis at the end of the line through me off originally because my brain immediately went to "method starts here"

Reply:
Going to cc @lyndsiWilliams here, just feel free to vote on a format, I'll just configure my IDE to use whichever. : D
```cs
/// 1
private AudioSource CreateAudioSource(string gameObjectName)
{
    return new GameObject(gameObjectName)
    {
        transform =
        {
            parent = transform
        }
    }.AddComponent<AudioSource>();
}


/// 2
private AudioSource CreateAudioSource(string gameObjectName) => new GameObject(gameObjectName) { transform = { parent = transform } }.AddComponent<AudioSource>(); }

/// 3
private AudioSource CreateAudioSource(string gameObjectName)
{
    return new GameObject(gameObjectName) { transform = { parent = transform } }.AddComponent<AudioSource>();
}
```

Reply:
Any of these are fine for me

Reply:
Went with `1` for now, just so this PR doesn't dangle around for too long. : D
Always open for a [Discussions thread](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/discussions) about it, If it's something we want to solidify further or change later down the line!